Procedures/Affiliated-Package-Review-Guidelines.md

```diff
@@ -0,0 +1,228 @@
+that have all-green scores may be featured in a separate table for
+well-maintained and supported packages.</td>
+</tr>
+</table>
+
+<!-- Once we figure out our package listing equivalent, reinstate this para.
+The document also includes ``monospaced keywords`` for the categories
+and levels.  These are the keywords and values to be used in the
+[registry.json](http://www.astropy.org/affiliated/registry.json)
+file that is the canonical source for affiliated package information.
+-->
+
+The categories in which we assess the package are the following:
+
+* Functionality (``'functionality'``)
+* Integration with the PlanetaryPy ecosystem  (``'ecointegration'``)
+* Documentation (``'documentation'``)
+* Testing (``'testing'``)
+* Development status (``'devstatus'``)
+* Python version compatibility (``'pythonver'``)
+
+### Functionality (`functionality`)
+
+We first need to make sure that the scope of the package is relevant
+for the affiliated package system. The scopes are:
+
+<table>
+<tr>
+<td width=150><img src="https://img.shields.io/badge/Out%20of%20scope-red.svg"
+                   alt="Out of scope"></td>
+<td>Not useful for planetary work, or specific to one project/collaboration.</td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/Specialized%20package-brightgreen.svg"
+         alt="Specialized package"></td>
+<td>Useful to planetary scientists and engineers working in a very
+specific domain/field, or with a specific instrument and
+usable not just by a single collaboration but anyone within
+that domain. <!-- Packages such as sncosmo fall into this category.--></td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/General%20package-brightgreen.svg"
+         alt="Green"></td>
+<td>Package that is useful for planetary researchers across more
+than just a single field or instrument. <!--Packages such as astroquery
+or astroplan fall into this category.--></td>
+</tr>
+</table>
+
+Note that general is not necessary better than specific, it’s just
+a way to make sure we can present these separately.
+
+### Integration with the PlanetaryPy ecosystem  (`ecointegration`)
+
+Next up, we need to check how well the package fits in to the existing PlanetaryPy
+ecosystem - does it make use of existing functionality, or does it duplicate it?
+
+<table>
+<tr>
+<td width=150><img src="https://img.shields.io/badge/Red-red.svg" alt="Red"></td>
+<td>Does not use PlanetaryPy or other affiliated packages anywhere
+where it should be possible, and/or uses other libraries instead.</td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/Orange-orange.svg" alt="Orange"></td>
+<td>Makes an effort to use PlanetaryPy or other affiliated packages in
+places, but still has other places where this could be done but
+isn’t</td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/Green-brightgreen.svg" alt="Green"></td>
+<td>Uses PlanetaryPy or other affiliated packages wherever possible.</td>
```

Comment:
`planetarypy`

Reply:
Same fix as above?

Reply:
yep

Reply:
but let me go through the rest before you start your engines for a change...

Reply:
Fari enough, I'll go do some other stuff and let you accumulate.